docs/docs/Components/components-tools.md

```diff
@@ -168,15 +168,16 @@ This component performs web searches using the [DuckDuckGo](https://www.duckduck
 
 | Name | Display Name | Info |
 |------|--------------|------|
-| input_value | Search Query | The search query to be used for the DuckDuckGo search |
-| max_results | Max Results | Maximum number of results to return |
-| max_snippet_length | Max Snippet Length | Maximum length of each result snippet |
+| input_value | Search Query | The search query to execute with DuckDuckGo. |
+| max_results | Max Results | The maximum number of search results to return. Default: `5`. |
+| max_snippet_length | Max Snippet Length | The maximum length of each result snippet. Default: `100`.|
 
 ### Outputs
 
 | Name | Display Name | Info |
 |------|--------------|------|
-| data | Data | List of search results as Data objects |
+| data | Data | List of search results as Data objects containing snippets and full content. |
```

Comment:
Why is data capitalized in the Info column?

Reply:
Added a link to the object